app/controllers/users_controller.rb

```diff
@@ -22,8 +22,24 @@ def dashboard
 
   def profile; end
 
+  def edit;  end
+  
   def update
+    update_user = BackEndFacade.update_user(users_params, current_user.id)
+    # require "pry"; binding.pry
+    if update_user[:errors].present?
+      flash[:error] = "Could not update your profile please try again!"
+      redirect_to profile_edit_path(current_user.id)
+    else
+      flash[:success] = "Successfully updated your profile!"
+      redirect_to profile_path(current_user.id)
+    end
     # TODO: write this method and create view
```

Comment:
Nice use of the errors to guide the conditional!  Can you please drop the comments below the method